esphome/config_validation.py

```diff
@@ -1347,15 +1347,17 @@ def extract_keys(schema):
 def typed_schema(schemas, **kwargs):
     """Create a schema that has a key to distinguish between schemas"""
     key = kwargs.pop("key", CONF_TYPE)
+    default_schema_option = kwargs.pop("default", None)
```

Comment:
```suggestion
    default_schema_option = kwargs.pop("default_type", None)
```

Might be confusing what `default` means here.